FaunaDB.Client/Query/Language.Miscellaneous.cs

```diff
@@ -421,5 +421,17 @@ public partial struct Language
         /// </summary>
         public static Expr ToDate(Expr expr) =>
             UnescapedObject.With("to_date", expr);
+
+        /// <summary>
+        ///   Merges two or more objects
+        /// </summary>
+        public static Expr Merge(Expr merge, Expr with) =>
+            UnescapedObject.With("merge", merge, "with", with);
+
+        /// <summary>
+        ///   Merges two or more objects
+        /// </summary>
+        public static Expr Merge(Expr merge, Expr with, Expr lambda) =>
+            UnescapedObject.With("merge", merge, "with", with, "lambda", lambda);
```

Comment:
please add an overload version that receives a native lambda `Func<Expr, Expr, Expr, Expr> lambda`

Reply:
nice one!!